docs/release_process_quarterly.md

```diff
@@ -0,0 +1,501 @@
+        - [ ] Open the jpo-ode project in an IDE
+        - [ ] Update the version number in the pom.xml files of the jpo-ode-common, jpo-ode-plugins, and jpo-ode-svcs projects to match the version number of the release. (e.g. 1.0.0)
+        - [ ] Update the dependencies in pom.xml files of the jpo-ode-common, jpo-ode-plugins, and jpo-ode-svcs projects to the version number set in the previous step. The `jpo-ode-plugins` depends on the `jpo-ode-common` project, and the `jpo-ode-svcs` project depends on the `jpo-ode-core` and `jpo-ode-plugins` projects. These should all be referencing the same version number. (e.g. 1.0.0)
+    - [ ] Update git submodule references for the ‘jpo-ode’ project to point to tagged commits in projects with updated `master` branches.
+        - [ ] Open the jpo-ode project in an IDE.
+        - [ ] Navigate to the asn1_codec directory and run `git checkout tags/asn1_codec-x.x.x` to update the submodule reference.
+        - [ ] Navigate to the jpo-cvdp directory and run `git checkout tags/jpo-cvdp-x.x.x` to update the submodule reference.
+        - [ ] Navigate to the jpo-security-svcs directory and run `git checkout tags/jpo-security-svcs-x.x.x` to update the submodule reference.
+        - [ ] Navigate to the jpo-sdw-depositor directory and run `git checkout tags/jpo-sdw-depositor-x.x.x` to update the submodule reference.
+        - [ ] Navigate to the jpo-utils directory and run `git checkout tags/jpo-utils-x.x.x` to update the submodule reference. 
+        - [ ] Commit these changes to the `release/(year)-(quarter)` branch & push the changes to the remote repository.
+        - [ ] Ensure these changes pass CI/CD checks before continuing.
+    - [ ] Merge `release/(year)-(quarter)` branch into `master` branch for the jpo-ode project, and create a release with the version number of the release. (e.g. jpo-ode-x.x.x)
+    - [ ] Create docker image
+        - [ ] Use the release for the jpo-ode to produce docker image with the same tag name, containing the version number.
+        - [ ] Upload docker image to [DockerHub](https://hub.docker.com/u/usdotjpoode)
+        - [ ] Tag docker image with the version number of the app. (e.g. 1.0.0)
+        - [ ] Tag docker image with year and quarter of release. (e.g. 2024-Q2)
+        - [ ] Tag docker image with 'latest' tag for the most recent release.
+    - [ ] Merge master branch into develop branch & verify that CI/CD passes.
+
+### 4. DockerHub Image Testing
+    - [ ] image starts up correctly
+    - [ ] http endpoint is reachable
+    - [ ] TIMs can be successfully pushed to http endpoint
+    - [ ] capable of ingesting messages via udp (see scripts in `scripts/tests` directory)
+        - [ ] TIMs
+        - [ ] BSMs
+        - [ ] SSMs
+        - [ ] SRMs
+        - [ ] SPaTs
+        - [ ] Maps
+        - [ ] PSMs
+
+
+## jpo-geojsonconverter
+### Prerequisites
+    - [ ] jpo-ode release completed
```

Comment:
Nature of dependencies:

- [ ] jpo-ode (Maven, Dockerhub) release completed

Reply:
Addressed!